tests/e2e/fixtures/statsFixtures.js

```diff
@@ -326,6 +321,179 @@ const seasonThreeMatches = [
   },
 ];
 
+const seasonOneGames = [
+  // Week 1
+  {
+    name: 'Clubs Week 1, Game 1',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs('2022-01-20').valueOf(),
+    p0: 'player1',
+    p1: 'player2',
+  },
+  {
+    name: 'Clubs Week 1, Game 2',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs('2022-01-20').valueOf(),
+    p0: 'player2',
+    p1: 'player3',
+  },
+  {
+    name: 'Clubs Week 1, Game 3',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs('2022-01-20').valueOf(),
+    p0: 'player1',
+    p1: 'player3',
+  },
+  {
+    name: 'Clubs Week 1, Game 4',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs('2022-01-20').valueOf(),
+    p0: 'player2',
+    p1: 'player3',
+  },
+
+  // Week 2
+  {
+    name: 'Clubs Week 2, Game 1',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs('2022-01-27').valueOf(),
+    p0: 'player1',
+    p1: 'player2',
+  },
+  {
+    name: 'Clubs Week 2, Game 2',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs('2022-01-27').valueOf(),
+    p0: 'player3',
+    p1: 'player4',
+  },
+  // Week 4 games
+  {
+    name: 'Clubs Week 2, Game 2',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs('2022-02-15').valueOf(),
+    p0: 'player3',
+    p1: 'player4',
+  },
+];
+
+const seasonTwoGames = [
+  // Week 1
+  {
+    name: 'Diamonds Week 1, Game 1',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs('2022-02-20').valueOf(),
```

Comment:
This makes the game in the 4th (shortened) week of Clubs 2022 which is what causes the crash without updated handling/initalization logic